scripts/ag

```diff
@@ -66,6 +66,7 @@ def make(db, force):
     make_settings_table()
     click.echo("Applying patches to database")
     patch_db(verbose=True)
+    migrate_procedures('public', 'ag', verbose=True)
```

Comment:
it only excepts for me if you run it outside of make (aka run it on a database that already had it run). That's why I moved it here.